### PR TITLE
updating code owners

### PR DESCRIPTION
**Pull request by dfry**

### CODE REVIEW

**Review by mdebarros — APPROVED** [review]
+1

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-*    james.bush@modusbox.com david.fry@modusbox.com juan.correa@modusbox.com matt.kingston@modusbox.com miguel.debarros@modusbox.com
+*    david.fry@modusbox.com juan.correa@modusbox.com yevhen.kyriukha@modusbox.com miguel.debarros@modusbox.com
```
Comment (kirgene) [inline CODEOWNERS:1]: Please split each user per line

**Review by kirgene — CHANGES_REQUESTED** [review]

**Review by kirgene — APPROVED** [review]

**Comment by dfry** [comment]
@mdebarros please approve

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**